pages/linux/scan-build.md

```diff
@@ -0,0 +1,35 @@
+# scan-build
+
+> scan-build is a command line utility that enables a user to run the
+> static analyzer over their codebase as part of performing a regular
+> build (from the command line).
```

Comment:
We don't split the command description. Why `command-line` twice?
```suggestion
> A command-line utility to run a static analyzer over a codebase as part of performing a regular build.
```

Reply:
Fixed :heavy_check_mark: